Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/IconOptionsView/IconOptionsViewModel.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+public protocol IconOptionsViewModel {
+    var imageName: String { get }
+    var title: GDSLocalisedString { get }
+    var body: GDSLocalisedString { get }
+    var contentViews: [UIView] { get }
```

Comment:
Minor naming thing here and this is a genuine question for discussion (I don't know what is better) but I think elsewhere we've called a similar thing a `childView` so perhaps this might be called `childViews`?